modules/custom/activity_creator/src/ActivityNotifications.php

```diff
@@ -100,38 +125,36 @@ public function markEntityNotificationsAsRead(AccountInterface $account, EntityB
    * @param \Drupal\Core\Entity\EntityBase $entity
    *   Entity object.
    *
-   * @throws \Drupal\Core\Entity\EntityStorageException
+   * @deprecated in opensocial:8.x-7.0 and is removed from opensocial:8.x-7.1. Use
```

Comment:
Are we sure this is removed in 8.x-7.1? Do we have another open PR to do this or is it scheduled?

Reply:
There's another `@deprecated` in this PR, same goes for that I suppose.

If we say something like that we need to schedule the work as well. I couldn't really figure out if that was the case.

Reply:
Also the removal-version should match standard: drupal:n.n.n or project:n.x-n.n